optlang/tests/test_cplex_interface.py

```diff
@@ -497,6 +499,98 @@ def test_qp_non_convex(self):
             model.optimize()
             self.assertAlmostEqual(model.objective.value, 2441.999999971)
 
+    class InfeasibleMIPTestCase(unittest.TestCase):
```

Comment:
I could drop those cases that are tested by accessing dual values in MIP already, true. However, I want to test here that all Cplex errors are adequately caught and reraised as `optlang.SolverError`s. That behavior is different between MIPs and LPs. Maybe I should extend this to LPs and QPs, too.